packages/mdc-dialog/test/component.test.ts

```diff
@@ -591,4 +591,15 @@ describe('MDCDialog', () => {
     component.layout();
     expect((component as any).foundation.layout).toHaveBeenCalled();
   });
+
+  it(`${strings.INITIAL_FOCUS_ATTRIBUTE} will focus when the dialog is opened`, () => {
```

Comment:
Button with `${strings.INITIAL_FOCUS_ATTRIBUTE} will be focused when the dialog is opened, with multiple initial focus buttons in DOM